src/Form/FieldSelect.js

```diff
@@ -0,0 +1,197 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+class FieldSelect extends React.Component {
+
+  handleChange(selectedValue) {
+    this.props.handleEvent('change',
+      this.props.name, selectedValue);
```

Comment:
I find it a little strange that I receive both html and id as data.
![](http://cl.ly/3F2W081R0Y0C/Image%202016-05-23%20at%2017.40.29.png)

Let's just return the `id` here